### PR TITLE
Feelsliketemp bugfix

### PR DESCRIPTION
**Pull request by MoseleyS**
Enforces a minimum wind speed for the wind chill calculation in the feels-like temperature plugin, as recommended by the paper this code is developed from.

Testing:
 - [x] Ran tests and they passed OK

### CODE REVIEW

**Comment by MoseleyS** [comment]
The KGO differences look like this:
image
As you would expect, a few points have got a little colder where the minimum wind speed has had an effect.

File: improver_tests/acceptance/SHA256SUMS [hunk 1]
```diff
@@ -249,7 +249,7 @@ b07166a4cd0e0842704aef550ad6398a54ad49cedc64c1fc2a7ef5e8b6d5aa61  ./feels_like_t
 236eecc75a83b7c6c6aab1530a0ca0e85371c44459f4a1830511eb91cc408def  ./feels_like_temp/ukvx/20181121T1200Z-PT0012H00M-relative_humidity_at_screen_level.nc
 e7d2d57da994abefcde41ec5df014d2368656baa68fceb81f45ee13ab1f67b36  ./feels_like_temp/ukvx/20181121T1200Z-PT0012H00M-temperature_at_screen_level.nc
 5a594e1f835b50088cfadde6e78e9af47232ccd6c628a15d991d4c8f0df52ef4  ./feels_like_temp/ukvx/20181121T1200Z-PT0012H00M-wind_speed_at_10m.nc
-a4abef4fa06612c7af9873a8a07fe8bfcff09631e95871d39f5d3026aa2771db  ./feels_like_temp/ukvx/kgo.nc
+8cc25b8c8dbea22abcb8765f248ac3f03094abe6078ee854786950dd2d2ab6bf  ./feels_like_temp/ukvx/kgo.nc
```
Comment (gavinevans) [inline improver_tests/acceptance/SHA256SUMS:252]: The acceptance test that produces this kgo is currently failing for me due to an attribute mismatch for the `mosg__model_configuration` attribute.

**Review by gavinevans — CHANGES_REQUESTED** [review]
Thanks @MoseleyS 👍 

I'm added one comment related to the acceptance test data, but otherwise this seems fine for avoiding anomalously high feels like temperatures at low wind speeds. I've visualised the contours below to better understand the behaviour.

Screenshot feels like temp

File: improver_tests/acceptance/SHA256SUMS (hunk 1 above)
Comment (MoseleyS) [reply improver_tests/acceptance/SHA256SUMS]: Yes, I'd forgotten an argument when I generated the KGO. Now corrected.

**Review by gavinevans — APPROVED** [review]

**Review by brhooper — APPROVED** [review]
This all looks fine

**Comment by MoseleyS** [comment]
- [x] Acceptance test branch merged

**Pull request merged**